WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormActionsFactory.swift

```diff
@@ -79,7 +80,8 @@ struct ProductFormActionsFactory: ProductFormActionsFactoryProtocol {
          isBundledProductsEnabled: Bool = ServiceLocator.featureFlagService.isFeatureFlagEnabled(.productBundles),
          isCompositeProductsEnabled: Bool = ServiceLocator.featureFlagService.isFeatureFlagEnabled(.compositeProducts),
          isMinMaxQuantitiesEnabled: Bool = ServiceLocator.featureFlagService.isFeatureFlagEnabled(.readOnlyMinMaxQuantities),
-         variationsPrice: VariationsPrice = .unknown) {
+         variationsPrice: VariationsPrice = .unknown,
+         editingSubscriptionEnabled: Bool = ServiceLocator.featureFlagService.isFeatureFlagEnabled(.subscriptionProducts)) {
```

Comment:
I think we should inject the feature flag service and use `MockFeatureFlagService` to test this. Looks like there is a TODO comment asking to remove the default parameters. 😄

Reply:
I thought injecting this parameter would make it easier to remove the legacy code when removing the feature flag. We would not need to update `MockFeatureFlagService` when adding a new flag and then revert when the flag is removed. This is helpful, especially since we are trying to follow the new process to remove flags immediately after releasing a new feature.